public/app/core/components/AppChrome/TopBar/TopNavBarMenu.tsx

```diff
@@ -31,11 +31,21 @@ export function TopNavBarMenu({ node }: TopNavBarMenuProps) {
       {node.children?.map((item) => {
         const translationKey = item.id && menuItemTranslations[item.id];
         const itemText = translationKey ? i18n._(translationKey) : item.text;
-
+        const showExternalLinkIcon = /^https?:\/\//.test(item.url || '');
         return !item.target && item.url?.startsWith('/') ? (
           <MenuItem url={item.url} label={itemText} key={item.id} />
         ) : (
-          <MenuItem onClick={() => onNavigate(item)} label={itemText} key={item.id} />
+          <MenuItem
+            onClick={() => onNavigate(item)}
```

Comment:
This is not great (CTRL + click does not work) and you don't see the URL when you hover over the link. Could not find where these help items are defined, would be great if we could make the ones that are links into normal links